modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPElementResourceImpl.java

```diff
@@ -265,7 +266,7 @@ public SXPElement postSXPElementCopy(Long sxpElementId) throws Exception {
 				contextUser),
 			_sxpElementService.addSXPElement(
 				sxpElement.getDescriptionMap(),
-				sxpElement.getElementDefinitionJSON(), false,
+				sxpElement.getElementDefinitionJSON(), null, false,
```

Comment:
Does this have/want to be null, couldn't we get the ERC from the Blueprint?

Reply:
It is a copy so we do not want to use the same ERC, that way it could return DuplicateSXP*ExternalReferenceCodeException. Using null it will generate a new ERC